etl/steps/data/garden/ophi/2024-10-28/multidimensional_poverty_index.meta.yml

```diff
@@ -201,7 +207,7 @@ tables:
           - "{definitions.description_key_assessments}"
           - "{definitions.description_key_weights}"
           - "{definitions.description_key_intensity}"
-          - "{definitions.description_key_flavor}"
+          - <% set cme_link = "https://ourworldindata.org/grapher/multidimensional-poverty-index-mpi" %> <% set hot_link = "https://ourworldindata.org/grapher/multidimensional-poverty-index-mpi-hot" %>"{definitions.description_key_flavor}"
```

Comment:
Hi @Marigold! Do you know how to avoid showing quotes in this bullet point? See the last bullet point [here](http://staging-site-mpi-add-dynamic-links/admin/charts/5948/edit):
<img width="673" alt="image" src="https://github.com/user-attachments/assets/54f2f53c-31a4-40f6-ab28-16196fad0a8d">

Reply:
i think it's now fixed

Reply:
It is! Thank you @lucasrodes! I thought this fix wasn't working before, maybe I've made a mistake then